pkg/wrangler/extapi.go

```diff
@@ -0,0 +1,126 @@
+package wrangler
+
+import (
+	"context"
+	"sync/atomic"
+
+	ext "github.com/rancher/rancher/pkg/generated/controllers/ext.cattle.io"
+	extv1 "github.com/rancher/rancher/pkg/generated/controllers/ext.cattle.io/v1"
+	wapiregv1 "github.com/rancher/wrangler/v3/pkg/generated/controllers/apiregistration.k8s.io/v1"
+	"github.com/rancher/wrangler/v3/pkg/generic"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/errors"
+	apiregv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
+)
+
+// EXTAPIContext is a scoped context which wraps the larger Wrangler context.
+// It includes EXT clients and factories which are initialized after the EXT
+// api-service is detected and established
+type EXTAPIContext struct {
+	*Context
+	Ext extv1.Interface
+	ext *ext.Factory
```

Comment:
Can we call these appropriately? Maybe
```suggestion
	Client extv1.Interface
	factory *ext.Factory
```
These are already part of type that starts with `EXT` so IMHO no need to repeat it again and again.

Reply:
Done with commit fb69e5d0d22fc6f17dc50695c3b8f876d1063d2c